report_generator/src/read_json.rs

```diff
@@ -0,0 +1,64 @@
+use serde::Deserialize;
+use serde::Serialize;
+use std::fs;
+use std::error::Error;
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Data {
```

Comment:
Nice typing, I think you generated these right? If so, could you leave a comment where? (For the next dev :) )

Reply:
Agree. A link would be nice.

Reply:
Also added the comment to the code